src/content/docs/logs/logs-context/configure-logs-context-nodejs.mdx

```diff
@@ -20,9 +20,182 @@ redirects:
 
-## Set up your Node.js app [#configure-node-js]
+* Cut through the noise of thousands of logs when troubleshooting time-critical issues, so you automatically see only the most relevant logs.
+* Navigate within multiple types of telemetry data, and have the data correlate back to the original issue.
+* Easily drill down into more detailed information from the same place in the UI.
+* Find the log lines that you need to identify and resolve a problem.
+
+For more information, including examples, learn how to get started with [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context).
+
+## Automatic logs in context options [#automatic]
+
+By enabling application logging, you can tell your APM agent to automatically collect and contextualize your logs. Application logging is not enabled by default at this time, but that may change in the future.
+
+To enable application logging, update your agent configuration:
+
+newrelic.js:
+
+```
+application_logging: {
+  enabled: true
+}
+```
+
+Environment variable:
+
+```
+NEW_RELIC_APPLICATION_LOGGING_ENABLED=true
+```
+
+You have three options to configure APM logs in context to send your app's logs and linking metadata automatically to New Relic.
+
+<CollapserGroup>
+  <Collapser
+    id="1-agent"
+    title="Option 1 (simplest): Let the agent decorate and forward your logs."
+  >
+    This is the simplest approach, and it's a great choice for developers who may not have the access or interest in setting up a log forwarder, or for accounts that want to see the power of logs and other linking metadata in context of their apps, without a lot of overhead.
+
+    Using this option, your logs will include `span.id`, `trace.id`, `hostname`, `entity.guid`, and `entity.name`. These attributes automatically link your logs to spans, traces, and other telemetry, making it easier to troubleshoot.
+
+    All you need to do is install an agent version with log forwarding capabilities ([NodeJS agent 8.11.0 or higher](https://docs.newrelic.com/docs/release-notes/agent-release-notes/nodejs-release-notes/)), and update your configuration to enable forwarding.
+
+    newrelic.js:
+
+    ```
+    application_logging: {
+      forwarding: {
+        enabled: true
+      }
+    }
+    ```
+
+    Environment variable:
+
+    ```
+    NEW_RELIC_APPLICATION_LOGGING_FORWARDING_ENABLED=true
+    ```
+
+    This option may be on by default in a future agent version.
+
+    **Optional adjustments:**
+
+    Once this is enabled, you also have control over the maximum number of logs sent to New Relic every minute. The default value is 10,000. If more than 10,000 logs are received in a 60-second window, your logs will begin to be sampled.
+
+    Set it to a higher number to receive more logs. Set it to a lower number to receive fewer logs. Any integer up to 100,000 is valid.
```

Comment:
We may want to leave out `Any integer up to 100,000 is valid` until we can confirm our performance is still ok. I put a comment here explaining this situation https://github.com/newrelic/node-newrelic/issues/1154#issuecomment-1119971331

Reply:
Ok I ran perf testing.  On the example app we were using it affected the CPU and memory but not to where increasing to 100k would be a serious problem

Reply:
I opted to include a brief warning about performance: "Increasing this value may increase the agent's utilization of CPU and RAM."